app/src/main/res/drawable/ic_build_state_created_16dp.xml

```diff
@@ -1,27 +1,36 @@
 <?xml version="1.0" encoding="utf-8"?>
-<vector
+<inset
```

Comment:
[Its used to adjust the bounds of the drawable](https://developer.android.com/reference/android/graphics/drawable/InsetDrawable.html), but here it is only used to wrap the vector so that it can be used independent of app:srcCompat. Detailed explanation can be found [here](https://medium.com/@chrisbanes/appcompat-v23-2-age-of-the-vectors-91cbafa87c88#.ebxcw236h).